neo/Helper.cs

```diff
@@ -139,6 +139,11 @@ public static Fixed8 Sum(this IEnumerable<Fixed8> source)
             return new Fixed8(sum);
         }
 
+        public static BigInteger Sum(this IEnumerable<BigInteger> source)
+        {
+            return source.Aggregate(BigInteger.Zero, (x, y) => x + y);
```

Comment:
is faster a regular foreach
![image](https://user-images.githubusercontent.com/3167973/57199184-6d2ffc80-6f7c-11e9-93f3-7e52fab2b6e0.png)

```suggestion
            var sum = BigInteger.Zero;
            foreach (var bi in source) sum += bi;
            return sum;
```